js/tests/unit/modal.spec.js

```diff
@@ -142,6 +142,30 @@ describe('Modal', () => {
       modal.toggle()
     })
 
+    it('should not adjust the inline margin and padding of sticky and fixed elements when element do not have full width', done => {
+      fixtureEl.innerHTML = [
+        '<div class="sticky-top" style="margin-right: 0px; padding-right: 0px; width: calc(100vw - 50%)"></div>',
```

Comment:
While debugging I found that the test was failed because the `clientWidth` of the `element` was greater than `window.innerWidth`. So I changed it to `width: calc(100vw - 50%)`. I am not sure about it, seems like a hack to me :smile: